diff --git a/node_modules/gantt-task-react/dist/index.js b/node_modules/gantt-task-react/dist/index.js
index 626bb39..43ed690 100644
--- a/node_modules/gantt-task-react/dist/index.js
+++ b/node_modules/gantt-task-react/dist/index.js
@@ -762,9 +762,9 @@ var Calendar = function Calendar(_ref) {
 
     for (var i = 0; i < dateSetup.dates.length; i++) {
       var date = dateSetup.dates[i];
-      var bottomValue = getLocaleMonth(date, locale);
+      var bottomValue = getDaysInMonth(date.getMonth(), date.getFullYear()).toString();
       bottomValues.push(React__default.createElement("text", {
-        key: bottomValue + date.getFullYear(),
+        key: date.getMonth() + "-" + date.getFullYear(),
         y: headerHeight * 0.8,
         x: columnWidth * i + columnWidth * 0.5,
         className: styles$5.calendarBottomText
@@ -810,7 +810,7 @@ var Calendar = function Calendar(_ref) {
         topValue = getLocaleMonth(date, locale) + ", " + date.getFullYear();
       }
 
-      var bottomValue = "W" + getWeekNumberISO8601(date);
+      var bottomValue = new Date(date.getFullYear(), date.getMonth(), date.getDate() + (7 - date.getDay()) % 7).getDate().toString();
       bottomValues.push(React__default.createElement("text", {
         key: date.getTime(),
         y: headerHeight * 0.8,
@@ -848,7 +848,7 @@ var Calendar = function Calendar(_ref) {
 
     for (var i = 0; i < dates.length; i++) {
       var date = dates[i];
-      var bottomValue = getLocalDayOfWeek(date, locale, "short") + ", " + date.getDate().toString();
+      var bottomValue = date.getDate().toString();
       bottomValues.push(React__default.createElement("text", {
         key: date.getTime(),
         y: headerHeight * 0.8,
diff --git a/node_modules/gantt-task-react/dist/index.modern.js b/node_modules/gantt-task-react/dist/index.modern.js
index b372269..52f07d6 100644
--- a/node_modules/gantt-task-react/dist/index.modern.js
+++ b/node_modules/gantt-task-react/dist/index.modern.js
@@ -761,9 +761,9 @@ var Calendar = function Calendar(_ref) {
 
     for (var i = 0; i < dateSetup.dates.length; i++) {
       var date = dateSetup.dates[i];
-      var bottomValue = getLocaleMonth(date, locale);
+      var bottomValue = getDaysInMonth(date.getMonth(), date.getFullYear()).toString();
       bottomValues.push(React.createElement("text", {
-        key: bottomValue + date.getFullYear(),
+        key: date.getMonth() + "-" + date.getFullYear(),
         y: headerHeight * 0.8,
         x: columnWidth * i + columnWidth * 0.5,
         className: styles$5.calendarBottomText
@@ -809,7 +809,7 @@ var Calendar = function Calendar(_ref) {
         topValue = getLocaleMonth(date, locale) + ", " + date.getFullYear();
       }
 
-      var bottomValue = "W" + getWeekNumberISO8601(date);
+      var bottomValue = new Date(date.getFullYear(), date.getMonth(), date.getDate() + (7 - date.getDay()) % 7).getDate().toString();
       bottomValues.push(React.createElement("text", {
         key: date.getTime(),
         y: headerHeight * 0.8,
@@ -847,7 +847,7 @@ var Calendar = function Calendar(_ref) {
 
     for (var i = 0; i < dates.length; i++) {
       var date = dates[i];
-      var bottomValue = getLocalDayOfWeek(date, locale, "short") + ", " + date.getDate().toString();
+      var bottomValue = date.getDate().toString();
       bottomValues.push(React.createElement("text", {
         key: date.getTime(),
         y: headerHeight * 0.8,
